chrome/extension/options.css

```diff
@@ -24,7 +24,7 @@ body {
   line-height: 1.4em;
   background: #f5f5f5;
   color: #4d4d4d;
-  width: 300px;
+  width: 320px;
```

Comment:
Yeah I agree that this belongs in #77, as that is also where I introduced it, and I'll further comment on it within that PR. I simply also added this PR already as I thought they were 2 different decisions to make, and I guess I was on a roll and wanted to submit both ;)